examples/terra-contract/tools/README.md

```diff
@@ -0,0 +1,113 @@
+# Tools
```

Comment:
This document should be the top-level README. This is what developers will want to see when they navigate to the package in github.

Reply:
I move it there.